src/scss/_themes.scss

```diff
@@ -0,0 +1,96 @@
+////
+/// @group o-card theme styles
+/// @link http://registry.origami.ft.com/components/o-card
+////
+
+/// Stand alone card theme styles
+///
+/// @param {String} $modifier [stand-alone] - the modifier classname you want to use
+@mixin oCardThemeStandAlone($modifier: 'stand-alone') {
+	&--#{$modifier} {
+		border-bottom: 1px solid oColorsGetColorFor(o-card-shadow, shadow);
+	}
+}
+
+/// Stand out card theme styles
+///
+/// @param {String} $modifier [standout] - the modifier classname you want to use
+@mixin oCardThemeStandOut($modifier: 'standout') {
+	&--#{$modifier} {
+		@include oColorsFor(o-card-standout, background);
+	}
+}
+
+/// Landscape card theme base styles
+/// modifies block elements tag, standfirst, timestamp, image, and heading
+/// to adapt to a landscape/horizontal layout
+///
+/// @param {String} $modifier [landscape] - the modifier classname you want to use
+@mixin oCardThemeLandscapeBase($modifier: 'landscape') {
+	&--#{$modifier} {
+		flex-direction: row;
+		flex-wrap: wrap;
+		justify-content: space-between;
+	}
+
+	&--#{$modifier} &__tag,
+	&--#{$modifier} &__standfirst,
+	&--#{$modifier} &__timestamp {
+		flex: 0 0 100%;
+	}
+
+	&--#{$modifier} &__heading {
+		width: 100%;
+		// Order third so it will appear after a left aligned image and
+		// before a right aligned image (see above)
+		order: 3;
+	}
+
+	&--#{$modifier} &__standfirst {
+		order: 4;
+	}
+
+	&--#{$modifier} &__timestamp {
+		width: 100%;
+		order: 5;
+	}
+}
+
+/// Landscape card with image styles
+/// modifies heading and image styles to adapt to a landscape/horizontal layout
+///
+/// @param {String} $modifier [landscape] - the modifier classname you want to use
+/// @param {String} $image-alignment [left] - alignment of an image in the landscape card
+@mixin oCardThemeLandscapeImage($modifier: 'landscape', $image-alignment: 'left') {
+
+	@if $image-alignment == 'right' {
+
+		&--#{$modifier}#{&}--image-right {
```

Comment:
It's not super readable, so a comment explaining what it´s doing would be helpful, maybe using the default values like

```
// .o-card--landscape.o-card--image-right